origin-js/src/contractInterface/users/v00_adapter.js

```diff
@@ -183,15 +185,31 @@ class V00_UsersAdapter {
     if (profileClaims.length) {
       const bytes32 = profileClaims[profileClaims.length - 1].data
       const ipfsHash = this.contractService.getIpfsHashFromBytes32(bytes32)
-      profile = await this.ipfsDataStore.load(PROFILE_DATA_TYPE, ipfsHash)
+      try {
+        profile = await this.ipfsDataStore.load(PROFILE_DATA_TYPE, ipfsHash)
+      } catch (error) {
+        console.error(`Can not read profile data from ipfs: ${error.message}`)
```

Comment:
Let's also add the ipfsHash to this error message to make it easier to troubleshoot.

Reply:
Good point, added it.